BunqSdk.Tests/Model/Generated/Object/NotificationUrlTest.cs

```diff
@@ -0,0 +1,228 @@
+        private const string GET_SCHEDULE_INSTANCE = "ScheduledInstance";
+        private const string GET_SHARE_INVITE_BANK_INQUIRY = "ShareInviteBankInquiry";
+        private const string GET_SHARE_INVITE_BANK_RESPONSE = "ShareInviteBankResponse";
+        
+        /// <summary>
+        /// Assert error constants.
+        /// </summary>
+        private const string ASSERT_SHOULD_NOT_REACH_THIS_CODE_ERROR = "Congratulations you\"ve reached unreachable code.";
+        private const string ASSERT_JSON_DECODE_ERROR = "Error accorded while decoding the JSON file.";
+        private const string ASSERT_OBJECT_IS_NULL_ERROR = "The field object of NotificationUrl is null.";
+        
+        /// <summary>
+        /// Model json paths constants.
+        /// </summary>
+        private const string BASE_PATH_JSON_MODEL = "../../../Resources/NotificationUrlJsons";
+        private const string JSON_PATH_MUTATION_MODEL = BASE_PATH_JSON_MODEL + "/Mutation.json";
+        private const string JSON_PATH_BUNQ_ME_TAB_MODEL = BASE_PATH_JSON_MODEL + "/BunqMeTab.json";
+        private const string JSON_PATH_CHAT_MESSAGE_ANNOUNCEMENT_MODEL = BASE_PATH_JSON_MODEL + 
+                                                                         "/ChatMessageAnnouncement.json";
+        private const string JSON_PATH_DRAFT_PAYMENT_MODEL = BASE_PATH_JSON_MODEL + "/DraftPayment.json";
+        private const string JSON_PATH_MASTER_CARD_ACTION_MODEL = BASE_PATH_JSON_MODEL + "/MasterCardAction.json";
+        private const string JSON_PATH_MONETARY_ACCOUNT_BANK_MODEL = BASE_PATH_JSON_MODEL + "/MonetaryAccountBank.json";
+        private const string JSON_PATH_PAYMENT_BATCH_MODEL = BASE_PATH_JSON_MODEL + "/PaymentBatch.json";
+        private const string JSON_PATH_REQUEST_INQUIRY_MODEL = BASE_PATH_JSON_MODEL + "/RequestInquiry.json";
+        private const string JSON_PATH_REQUEST_RESPONSE_MODEL = BASE_PATH_JSON_MODEL + "/RequestResponse.json";
+        private const string JSON_PATH_SCHEDULE_PAYMENT_MODEL = BASE_PATH_JSON_MODEL + "/ScheduledPayment.json";
+        private const string JSON_PATH_SCHEDULE_INSTANCE_MODEL = BASE_PATH_JSON_MODEL + "/ScheduledInstance.json";
+        private const string JSON_PATH_SHARE_INVITE_BANK_INQUIRY_MODEL = BASE_PATH_JSON_MODEL + 
+                                                                         "/ShareInviteBankInquiry.json";
+
+        private const string JSON_PATH_SHARE_INVITE_BANK_RESPONSE_MODEL = BASE_PATH_JSON_MODEL +
+                                                                          "/ShareInviteBankResponse.json";
+        
+        /// <summary>
+        /// Model root key.
+        /// </summary>
+        private const string KEY_NOTIFICATION_URL_MODEL = "NotificationUrl";
+
+        private void ExeceuteNotificationUrlTest(
+            string expectedJsonFileName,
+            Type classNameExpected,
+            string referencedObjectGetterName)
```

Comment:
I believe we want to have the closing brace on a new line, such as
```
    string referencedObjectGetterName
) {
```

Reply:
Also, the name of the argument would be different: `referencedObjectPropertyName` rather than `referencedObjectGetterName` (since we are dealing with properties in C#).